recipes/gumbo/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "libgumbo" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  - url: https://github.com/google/gumbo-parser/archive/v{{ version }}.tar.gz
+    sha256: 28463053d44a5dfbc4b77bcf49c8cee119338ffa636cc17fc3378421d714efad
+  - path: CMakeLists.txt  # [win]
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libgumbo', max_pin='x.x') }}
+
+requirements:
+  build:
+    - "{{ compiler('c') }}"
+    - "{{ compiler('cxx') }}"
```

Comment:
It claims to be pure C99 in the description?
```suggestion
```

Reply:
agree, however autotools fails if it's not finding a C++ compiler.